src/test/java/itm/oss/splitter/SettlementTest.java

```diff
@@ -0,0 +1,33 @@
+package itm.oss.splitter;
+
+import org.junit.jupiter.api.Test;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class SettlementTest {
+
+    @Test
+    void simpleTest() {
```

Comment:
The function name `simpleTest` is too simple and doesn't show the expected outcome of the test.

Renaming name following the 'test_whenCondition_thenExpectedResult' pattern would clearly state the condition and the expected outcome. This helps reviewers understand the test's purpose instantly.

I suggest renaming it to:

```suggestion
void test_singlePayment_whenOnePersonOwesAnother() {

Reply:
Thank you for your helpful suggestion! I agree that renaming the method makes it clearer, so I’ll apply the change.

Reply:
Thanks for your addressing that. It looks good to me!